portal/src/graphql/portal/SingleSignOnConfigurationScreen.tsx

```diff
@@ -428,10 +428,15 @@ const SingleSignOnConfigurationWidgetWrapper: React.FC<WidgetWrapperProps> = fun
       : "";
   }, [providerIndex]);
 
+  const clientSecretJsonPointer = useMemo(() => {
+    return `/secrets/sso.oauth.client/${providerType}`;
```

Comment:
In backend this is `/secrets/sso.oauth.client/{alias}`, `alias` might not be the same as `providerType`